2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Configure stats pid to 679932
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Loading settings from /home/user126/.config/wandb/settings
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Loading settings from /home/user126/TFformer/wandb/settings
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home/user126/TFformer/main.py', 'program': '/home/user126/TFformer/main.py'}
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:_log_setup():524] Logging user logs to ./wandb/run-20240402_142750-d6l8jt0b/logs/debug.log
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:_log_setup():525] Logging internal logs to ./wandb/run-20240402_142750-d6l8jt0b/logs/debug-internal.log
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:init():564] calling init triggers
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:init():614] starting backend
2024-04-02 14:27:50,655 INFO    MainThread:679932 [wandb_init.py:init():618] setting up manager
2024-04-02 14:27:50,656 INFO    MainThread:679932 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-02 14:27:50,657 INFO    MainThread:679932 [wandb_init.py:init():624] backend started and connected
2024-04-02 14:27:50,661 INFO    MainThread:679932 [wandb_init.py:init():716] updated telemetry
2024-04-02 14:27:50,666 INFO    MainThread:679932 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-04-02 14:27:51,499 INFO    MainThread:679932 [wandb_run.py:_on_init():2254] communicating current version
2024-04-02 14:27:51,604 INFO    MainThread:679932 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-02 14:27:51,604 INFO    MainThread:679932 [wandb_init.py:init():800] starting run threads in backend
2024-04-02 14:27:54,053 INFO    MainThread:679932 [wandb_run.py:_console_start():2233] atexit reg
2024-04-02 14:27:54,053 INFO    MainThread:679932 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-04-02 14:27:54,053 INFO    MainThread:679932 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-04-02 14:27:54,053 INFO    MainThread:679932 [wandb_run.py:_redirect():2178] Redirects installed.
2024-04-02 14:27:54,054 INFO    MainThread:679932 [wandb_init.py:init():841] run started, returning control to user process
2024-04-02 14:27:54,103 INFO    MainThread:679932 [wandb_run.py:_config_callback():1342] config_cb None None {'dim_A': 4, 'dim_X': 0, 'dim_Y': 1, 'dim_V': 1, 'seq_hidden_units': 16, 'num_heads': 2, 'head_size': None, 'dropout_rate': 0.1, 'num_layer': 1}
2024-04-02 14:28:02,656 INFO    MainThread:679932 [wandb_run.py:_config_callback():1342] config_cb None None {'dim_A': 4, 'dim_X': 0, 'dim_Y': 1, 'dim_V': 1, 'seq_hidden_units': 16, 'num_heads': 2, 'head_size': None, 'dropout_rate': 0.1, 'num_layer': 1}
2024-04-02 14:28:03,036 INFO    MainThread:679932 [wandb_run.py:_config_callback():1342] config_cb None None {'dim_A': 4, 'dim_X': 0, 'dim_Y': 1, 'dim_V': 1, 'seq_hidden_units': 16, 'num_heads': 2, 'head_size': None, 'dropout_rate': 0.1, 'num_layer': 1}
2024-04-02 14:28:10,035 WARNING MsgRouterThr:679932 [router.py:message_loop():77] message_loop has been closed
